node.js/week2/homework/warmup/index.js

```diff
@@ -1,6 +1,40 @@
 const express = require("express");
 const app = express();
 
+  
+
 app.get("/", (req, res) => res.send("nodejs week2 homework"));
 
+app.get("/numbers/add", (request, response) => {
+    console.log(request.query);
+
+    const firstNumber = parseFloat(request.query.first);
+    const secondNumber = parseFloat(request.query.second);
+    const sum = firstNumber + secondNumber;
+
+    if (isNaN(sum)) {
+        response.status(404).json({ error: "Input must be number" });
```

Comment:
// 400 Bad Request maybe? 
```suggestion
        response.status(400).json({ error: "Input must be number" });
```